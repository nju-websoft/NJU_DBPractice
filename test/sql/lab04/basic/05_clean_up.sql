open database db2025;
drop table stock;
show index;
exit;