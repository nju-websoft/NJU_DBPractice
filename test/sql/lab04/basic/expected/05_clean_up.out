
+--------------+--------------+--------------+--------------+--------------+--------------+
| Database     | Table        | FieldNum     | RecordLength | StorageModel | IndexNum     | 
|              |              |              |              |              |              | 
+--------------+--------------+--------------+--------------+--------------+--------------+
| db2025       | stock        | 17           | 114          | NARY_MODEL   | 8            | 
+--------------+--------------+--------------+--------------+--------------+--------------+
Total tuple(s): 1

+--------------+--------------+--------------+--------------+--------------+--------------+
| Database     | Table        | Index        | IndexID      | IndexType    | KeySchema    | 
+--------------+--------------+--------------+--------------+--------------+--------------+
Total tuple(s): 0
